### ISSUE TITLE
[skip ci] is not respected for put

### ISSUE THREAD

**Issue by giner** [concourse/git-resource#196]
Commits with `[skip ci]` should not update version of the resource if disable_ci_skip is enabled.

In case of pipeline like this:
`gitrepo_resource -> GET -> something_doing_job -> PUT -> gitrepo_resource`
The PUT updates the version and `something_doing_job` is triggered again.

There is a workaround described in #18, however this makes pipeline messy and harder to read.

**Comment by Yacine-A** [comment]
Is anyone working on this issue ?

**Comment by kalfa** [comment]
+1 for this.

create another resource for that is also a waste, since it requires a further running container for each resource modified for each pipeline defined.

nowadays it's a fairly common practice to have pipelines to update a version-pinning file (e.g. Pipfile.lock in pipenv/python world), I wouldn't consider it a corner use case.

would this example work? https://stackoverflow.com/questions/42607033/how-to-commit-changes-to-git-in-concourse-build
it doesn't require a new resource, which while is still unglish, is a better solution than #18.

**Comment by vito** [comment]
This issue is impossible to fix in the `git` resource, which is why #18 was closed (though in hindsight the reason for closing could have been clearer). It's a design problem in Concourse which we're tackling as part of concourse/rcfcs#1 (and tangentially concourse/rfcs#11).

I've opened https://github.com/concourse/concourse/issues/3463 so there's something more concrete to point to, but I'm going to close this again as there's nothing we can do on the `git` resource's side for this.
